tests/test_jams.py

```diff
@@ -1117,3 +1117,9 @@ def _foo():
 
         # And that it says the right thing (roughly)
         assert 'deprecated' in str(out[0].message).lower()
+
+
+def test_numpy_serialize():
```

Comment:
This looks good for `JObject`, which I guess is testing `serialize_object` under the hood ... I'm wondering if it's worth checking the float casting for the observation data in `Annotation`?